test/example/target.js

```diff
@@ -479,6 +480,7 @@ class Wrap extends React.Component {
           <PropTester {...complexProps}/>
           <PropTester {...uninspectableProps}/>
           <PropTester massiveMap={massiveMap}/>
+          <PropTesterWithPropTypes validProp="test" invalidProp="test" propWithoutPropTypes="test" />
```

Comment:
This is my concern here - it would be handy to have an invalid prop here for testing purposes, and it causes a propType error (duh). What do you think? Should I remove it for now?